hw/dv/sv/cip_lib/doc/index.md

```diff
@@ -85,8 +85,8 @@ virtual function void initialize(bit [31:0] csr_base_addr = '1);
   super.initialize(csr_base_addr); // ral model is created in `super.initialize`
   tl_intg_alert_fields[ral.a_status_reg.a_field] = value;
 ```
-* **en_tl_intg_gen**: Controls whether the TLUL integrity error is generated in
-  sequences / checked by the scoreboard.
+* **tl_intg_gen_type**: Controls what the TLUL integrity is generated in
```

Comment:
what -> whether?

Reply:
this variable has been removed. Removed this line as well.